guides/v2.2/config-guide/varnish/config-varnish-configure.md

```diff
@@ -263,6 +265,62 @@ Via: 1.1 varnish-v4
 X-Magento-Cache-Debug: HIT
 ```
 
+## Backend Fetch Failed errors
+
+If the length of [cache](https://glossary.magento.com/cache) tags used by Magento exceed Varnish's default of 8192 bytes, you can see HTTP 503 (Backend Fetch Failed) errors in the browser. The errors might display similar to the following:
+
+```terminal
+Error 503 Backend fetch failed
+Backend fetch failed
+```
+
+To resolve this issue, increase the default value of the `http_resp_hdr_len` parameter in your Varnish configuration file. The `http_resp_hdr_len` parameter specifies the max header length _within_ the total default response size of 323768 bytes.
+
+{:.bs-callout .bs-callout-info}
+If the `http_resp_hdr_len` value exceeds 32K, you must also increase the default response size using the `http_resp_size` parameter.
+
+1.	As a user with `root` privileges, open your Vanish configuration file in a text editor:
+
+	*	CentOS 6: `/etc/sysconfig/varnish`
+	*	CentOS 7: `/etc/varnish/varnish.params`
+	*	Debian: `/etc/default/varnish`
+	*	Ubuntu: `/etc/default/varnish`
+
+2.	Search for the `http_resp_hdr_len` parameter.
+3.	If the parameter doesn't exist, add it after `thread_pool_max`.
+4.	Set `http_resp_hdr_len` to a value equal to the product count of your largest [category](https://glossary.magento.com/category) multiplied by 21. (Each product tag is about 21 characters in length.)
+
+    For example, setting the value to 65536 bytes should work if your largest category has 3,000 products.
+
+    For example:
```

Comment:
Two "for example" in a row. Can this be combined or are they separate thoughts?

Reply:
I think the second occurrence can be deleted actually. However, since Support has chosen to migrate the original troubleshooting topic to the KB, I need to revert the changes that move this content into this topic.